sentry-kotlin-multiplatform/build.gradle.kts

```diff
@@ -216,3 +231,25 @@ buildkonfig {
         buildConfigField(STRING, "SENTRY_COCOA_VERSION", Config.Libs.sentryCocoaVersion)
     }
 }
+
+private fun KotlinMultiplatformExtension.addNoOpTargets() {
+    js(IR) {
```

Comment:
oh, should we use BOTH here? https://kotlinlang.org/docs/js-ir-compiler.html#authoring-libraries-for-the-ir-compiler-with-backwards-compatibility